### PR TITLE
fix timeout val

### PR DESCRIPTION
**Pull request by BelAn97**
waits for UIElement
fix getSearchContext if parent not present
get list of attributes for WebList, ListBase
tests for values and attrs

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4842347&bid=16076429)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/combobox/DataListTests.java [hunk 1]
```diff
@@ -22,7 +22,8 @@ public void before() {
         html5Page.shouldBeOpened();
         iceCream.select(text);
     }
-    String text = "Coconut";
+    private String text = "Coconut";
+    private String[] values = {"Chocolate", "Coconut", "Mint", "Strawberry", "Vanilla"};
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/combobox/DataListTests.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4842347)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4842347&bid=16076429)

**Comment by mf-code** [comment]
[4512] - https://transfer.sh/oI5eK/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4512] - https://transfer.sh/38MHK/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4512] - https://transfer.sh/9jTK4/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4512] - Allure report on Netlify: https://5e21f26869a49e01ca22843f--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4514] - https://transfer.sh/mB8eE/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4514] - https://transfer.sh/1exdN/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4514] - https://transfer.sh/kIlBU/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4514] - Allure report on Netlify: https://5e21f9525fb61801a70e52e7--musing-swartz-6fc060.netlify.com

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4842347&bid=16076429)

**Comment by RomanIovlev** [comment]
Thanks, good pull request

**Pull request merged**